### PR TITLE
Nerfs Cargo - No more Factorio Simulator Free money exploit 2023

### PR DESCRIPTION
**Pull request by JamieD1**
Too many people are abusing the autolathes to get free money, this causes lags and is just not a viable solution.

:cl:  
tweak: Autolathed Items can no longer be sold for profit.
/:cl:

### CODE REVIEW

**Comment by Moltijoe** [comment]
rip factorio

File: code/game/machinery/autolathe.dm [hunk 1]
```diff
@@ -365,6 +365,7 @@
 				for(var/i=1, i<=multiplier, i++)
 					var/obj/item/new_item = new D.build_path(A)
 					new_item.materials = new_item.materials.Copy()
+					new_items.autolathed = TRUE
```
Comment (BurgerLUA) [inline code/game/machinery/autolathe.dm]: ``new_item.autolathed = TRUE``

**Review by BurgerLUA — CHANGES_REQUESTED** [review]

File: code/game/machinery/autolathe.dm (hunk 1 above)
Comment (BurgerLUA) [reply code/game/machinery/autolathe.dm]: how do i suggest code edits help this is hard

**Comment by ReddicusDragon** [comment]
new_items.autolathed var doesn't exist :(

File: code/game/machinery/autolathe.dm (hunk 1 above)
Comment (tattax) [reply code/game/machinery/autolathe.dm]: click the plus icon on the side

Comment (tattax) [reply code/game/machinery/autolathe.dm]: then click the little file thingy

**Comment by JamieD1** [comment]
oopsie

**Comment by ReddicusDragon** [comment]
if this fails again I'm slapping on the closing after 24h tag

**Comment by JamieD1** [comment]
> if this fails again I'm slapping on the closing after 24h tag

Dont do it

**Review by adamsong — APPROVED** [review]
Code works fine, solution feels like a shortcut, but eh

File: code/game/objects/items.dm [hunk 2]
```diff
@@ -143,6 +143,8 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 	/// Is this item fryable without a syndicate frying pan
 	var/fryable = FALSE
 
+	var/printed = FALSE
```
Comment (ghost) [inline code/game/objects/items.dm:146]: Maybe just add a trait for that?

Comment (JamieD1) [reply code/game/objects/items.dm:146]: > Maybe just add a trait for that?

You mean an ITEM/OBJ Flag?

**Comment by JamieD1** [comment]
> Code works fine, solution feels like a shortcut, but eh

It is, honestly was the easiest solution I could think of

File: code/game/objects/items.dm (hunk 2 above)
Comment (ghost) [reply code/game/objects/items.dm:146]: That one that is added through ADD_TRAIT

**Comment by Moja10** [comment]
Exporting autolathe items has been a thing for forever, its literally the most poverty way to make money. People wouldn't have to do it if literally every single other method wasn't nerfed to dirt. Mining exports, gas exports, bounties, and export prices have all been repeatedly slashed to the point where spamming 2000 fire extinguishers is now the most profitable thing. Do you see how the problem isn't the fire extinguishers, but the several combined changes that lead us down this accursed path? This was literally never a problem until the nerfs get so bad that spamming garbage is more profitable than any of the actual efforted exports are.

File: code/game/objects/items.dm (hunk 2 above)
Comment (Bibby0110) [reply code/game/objects/items.dm:146]: do that

**Review by BurgerLUA — APPROVED** [review]
should be a flag but other than that it's gucci

File: code/game/objects/items.dm (hunk 2 above)
Comment (Moltijoe) [reply code/game/objects/items.dm:146]: yeah, probably best that not every item has an additional variable to track

**Comment by JamieD1** [comment]
> Exporting autolathe items has been a thing for forever, its literally the most poverty way to make money. People wouldn't have to do it if literally every single other method wasn't nerfed to dirt. Mining exports, gas exports, bounties, and export prices have all been repeatedly slashed to the point where spamming 2000 fire extinguishers is now the most profitable thing. Do you see how the problem isn't the fire extinguishers, but the several combined changes that lead us down this accursed path? This was literally never a problem until the nerfs get so bad that spamming garbage is more profitable than any of the actual efforted exports are.

Its a problem because I made the autolathe automatic.

Lags the server, makes garbage subsystem eat shit.

**Pull request merged**